.gitlab-ci.yml

```diff
@@ -149,96 +149,96 @@ serdiv-quest:
     - build
 
 ###################################
-# tests with serpent cache system
+# tests with WB cache system
 
 s-asm-quest:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-asm-tests defines=PITON_ARIANE+AXI64_CACHE_PORTS batch-mode=1
+    - make -j${NUM_JOBS} run-asm-tests defines=WT_DCACHE batch-mode=1
   dependencies:
     - build
 
 # atomics
 s-amo-quest:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-amo-tests defines=PITON_ARIANE+AXI64_CACHE_PORTS batch-mode=1
+    - make -j${NUM_JOBS} run-amo-tests defines=WT_DCACHE batch-mode=1
   dependencies:
     - build
 
-# floating point    
+# floating point
 s-fp-quest:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-fp-tests defines=PITON_ARIANE+AXI64_CACHE_PORTS batch-mode=1
+    - make -j${NUM_JOBS} run-fp-tests defines=WT_DCACHE batch-mode=1
   dependencies:
     - build
 
 s-bench-quest:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-benchmarks defines=PITON_ARIANE+AXI64_CACHE_PORTS batch-mode=1
+    - make -j${NUM_JOBS} run-benchmarks defines=WT_DCACHE batch-mode=1
   dependencies:
     - build
 
 # rv64ui-p-* tests
 s-asm1-ver:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-asm-tests1-verilator defines=PITON_ARIANE+AXI64_CACHE_PORTS
+    - make -j${NUM_JOBS} run-asm-tests1-verilator defines=WT_DCACHE
   dependencies:
     - build
 
 # rv64ui-v-* tests
 s-asm2-ver:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-asm-tests2-verilator defines=PITON_ARIANE+AXI64_CACHE_PORTS
+    - make -j${NUM_JOBS} run-asm-tests2-verilator defines=WT_DCACHE
   dependencies:
     - build
 
 # rv64um-*-* tests
 mul-ver:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-mul-verilator defines=PITON_ARIANE+AXI64_CACHE_PORTS
+    - make -j${NUM_JOBS} run-mul-verilator defines=WT_DCACHE
   dependencies:
     - build
 
 # atomics
 amo-ver:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-amo-verilator defines=PITON_ARIANE+AXI64_CACHE_PORTS
+    - make -j${NUM_JOBS} run-amo-verilator defines=WT_DCACHE
   dependencies:
     - build
 
 # floating point
 s-fp-ver:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-fp-verilator defines=PITON_ARIANE+AXI64_CACHE_PORTS
+    - make -j${NUM_JOBS} run-fp-verilator defines=WT_DCACHE
   dependencies:
     - build
 
 s-bench-ver:
-  stage: serpent
+  stage: wb
   script:
-    - make -j${NUM_JOBS} run-benchmarks-verilator defines=PITON_ARIANE+AXI64_CACHE_PORTS
+    - make -j${NUM_JOBS} run-benchmarks-verilator defines=WT_DCACHE
   dependencies:
     - build
 
 s-icache-quest:
-  stage: serpent
+  stage: wb
   script:
-    - cd tb/tb_serpent_icache/
+    - cd tb/tb_wb_icache/
```

Comment:
should be `tb_wt_icache`, no?

Reply:
or we can just leave the wb/wt identifier away in case of the icache, since wt and wb are irrelevant

Reply:
Yeah, I was thinking about similar things but then decided to go for `wt_icache` but the `wt` flavor of icache is used in the `wt` cache subsystem. In a next step we could align just to use one icache e.g.: the `wt` but I'd prefer to defer that decision a bit.

Reply:
Ok SGTM.